src/client/routes.tsx

```diff
@@ -37,6 +37,9 @@ export const GatewayRoutes = () => (
     <Route exact path={Routes.REGISTRATION}>
       <RegistrationPage />
     </Route>
+    <Route exact path={Routes.REGISTRATION_EMAIL_SENT}>
+      <EmailSentPage />
```

Comment:
We could just pass in `subTitle` here? Unless I'm missing a use case (possible!)

Reply:
Or just have `RegistrationSentPage` and then we can hardcode these dynamic values? (`subTitle`, `previousPage` and `pageTitle` - non of them really need to be set in state)

Reply:
yes I like that idea to have a `RegistrationSentPage` :) I'm going to make a small change anyway to POST the resend email to a different endpoint to the "change email" link so I'll do that as part of this change